sim/network.go

```diff
@@ -57,7 +60,7 @@ func NewNetwork(latency LatencyModel, traceLevel int) *Network {
 		participants:               map[f3.ActorID]f3.Receiver{},
 		participantIDs:             []f3.ActorID{},
 		queue:                      messageQueue{},
-		clock:                      0,
+		clock:                      time.Now(),
```

Comment:
Please continue to use zero, or some zero-like value, rather than something that depends on the external world.